Model/Classes/Flight.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AirportTicketBooking.Utils;
+using System.Text;
+
+namespace AirportTicketBooking.Model.Classes;
+#nullable disable
+public class Flight
+{
+    private static long _id = 0;
+    public long Id { get; init; } = ++_id;
+    public Airport DepartureAirport { get; set; }
+    public Airport DestinationAirport { get; set; }
+    public Country DepartureCountry { get; set; }
+    public Country DestinationCountry { get; set; }
+    public DateTime Time { get; set; }
```

Comment:
If this is the time then we should use `TimeSpan` type, but if you want to store the date then this property should be renamed